installers/python-ci/radiasoft-download.sh

```diff
@@ -51,7 +51,7 @@ python_ci_main() {
             if [[ -f test.sh ]]; then
                 bash test.sh
             else
-                pykern ci run
+                pykern ci run --github-ci-repo=$GITHUB_REPOSITORY
```

Comment:
This should just use the environment variable and not need an extra command line arg.